jfr-streaming/src/test/java/io/opentelemetry/contrib/jfr/streaming/JfrTelemetryTest.java

```diff
@@ -45,7 +44,16 @@ void create_Default() {
       assertThat(logs.getEvents()).hasSize(1);
       logs.assertContains("Starting JfrTelemetry");
 
-      await().untilAsserted(() -> assertThat(reader.collectAllMetrics()).isNotEmpty());
+      assertThat(reader.collectAllMetrics())
+          .isNotEmpty()
+          .allSatisfy(
+              metric -> {
+                assertThat(metric.getInstrumentationScopeInfo().getName())
+                    .isEqualTo("io.opentelemetry.contrib.jfr");
```

Comment:
maybe, to match the module name?

```suggestion
                    .isEqualTo("io.opentelemetry.contrib.jfr.streaming");
```